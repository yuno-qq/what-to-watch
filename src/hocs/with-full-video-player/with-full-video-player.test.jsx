import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"

import withFullVideoPlayer from "./with-full-video-player.jsx"


describe(`withFullVideoPlayer() snapshots:`, () => {
  test(`should render correctly`, () => {
    const imageSrc = `image.svg`
    const videoSrc = `movie.avv`

    const MockComponentWrapped = withFullVideoPlayer((props) => <div className="MOCK_COMPONENT">{props.renderItem(imageSrc, videoSrc)}</div>)
    const tree = renderer.create(<MockComponentWrapped />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
