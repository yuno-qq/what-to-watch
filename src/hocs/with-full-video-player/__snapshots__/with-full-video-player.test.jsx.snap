// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withFullVideoPlayer() snapshots: should render correctly 1`] = `
<div
  className="MOCK_COMPONENT"
>
  <video
    className="player__video"
    loop={false}
    muted={false}
    onCanPlay={[Function]}
    onLoadedMetadata={[Function]}
    onPause={[Function]}
    onPlay={[Function]}
    onTimeUpdate={[Function]}
    onWaiting={[Function]}
    poster="image.svg"
    src="movie.avv"
  />
</div>
`;
